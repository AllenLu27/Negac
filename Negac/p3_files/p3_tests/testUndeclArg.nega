int callee(){

}

int fn(int a){
	    callee(1,1,1,1,1,1,b);
}
