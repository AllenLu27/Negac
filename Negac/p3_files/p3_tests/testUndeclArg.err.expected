FATAL [6,25]: Undeclared identifier
FATAL [6,6]: Function call with wrong number of args
Semantic analysis failed
